JS/Lesson34/gulcin/recipe-app/src/modules/recipes/RecipeDetails/styles.css

```diff
@@ -0,0 +1,66 @@
+h2 {
+    font-size: 30px;
+    color: #000;
+    position: relative;
+}
+
+h3 {
+    letter-spacing: 3px;
+    font-size: 1.3em;
+    border-bottom: 1px solid #535252;
+    padding-bottom: 8px;
+}
+
+img {
+    width: 100%;
+    height: 100%;
+    border-radius: 7px;
+
+}
+
+ul {
+    padding: 0;
+}
+
+.recipe-details {
+    border-top: 3px solid #acadb455;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.img-ingredients {
+    padding: 15px 35px;
+    ;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 25px;
+    width: 60%;
+    border-radius: 4px;
+    background-color: #e5e5f7;
+    background-image: repeating-radial-gradient(circle at 0 0, transparent 0, #e5e5f7 8px), repeating-linear-gradient(#acadb455, #acadb4);
+    font-size: 1.1em;
+
+}
+
+.recipe-ingredients {
+    width: 50%;
```

Comment:
It looks quite small on mobile, it would be good to put some more work into making the application look good on all kinds of devices.
Feel free to do it later, no need to add to this assignment.